cnf-certification-test/manageability/suite.go

```diff
@@ -57,3 +65,29 @@ func testContainersImageTag(env *provider.TestEnvironment) {
 		ginkgo.Fail("Containers have been found that are missing tag(s) on image.")
 	}
 }
+
+// The name field in the ContainerPort section must be of the form <protocol>[-<suffix>] where <protocol> is one of the following,
+// and the optional <suffix> can be chosen by the application. Allowed protocol names: grpc, grpc-web, http, http2, tcp, udp.
+var allowedProtocolNames = map[string]bool{"grpc": true, "http": true, "http2": true, "tcp": true, "udp": true}
+
+func containerPortNameFormatCheck(portName string) bool {
```

Comment:
Just for organization, should we move this to something like `manageability/ports.go` or something similar?  To keep the suite.go clear of helper funcs?

Reply:
This can be done in a different PR by someone else.